RSP_Requirements_Design_Document/index.html

```diff
@@ -256,9 +256,10 @@
               (application, validity, transactional). 
               In case no timestamp is associated to an RDF stream data item, the system is responsible of managing 
               time-based ordering of stream items.</li>
-          <li>RSPs should process streams of data actively and in-stream, without the need of storing them. 
+          <li>RSPs should be capable of processing streams of data reactively.</li>
```

Comment:
What does "reactively" means? I think that it's a bit underdefined. Are we going to associate QoS requirements to the queries?